core/src/main/scala/org/apache/spark/rdd/CoalescedRDDPublic.scala

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
I tried to follow the naming convention for other classes, let me know if you still want lower-case.